server/db/models/orders.js

```diff
@@ -0,0 +1,24 @@
+// define Orders model here
+const Sequelize = require('sequelize');
+const db = require('./../database.js');
+
+const { UUID, UUIDV4, STRING, DECIMAL } = Sequelize;
+
+const Order = db.define('orders', {
+    id: {
+        primaryKey: true,
+        type: UUID,
+        defaultValue: UUIDV4,
+    },
+    orderCost: {
+        //how to make order cost as the sum
```

Comment:
Industry standard is to prepend comments like this with `TODO` as editors support some features that makes it easier not to forget these!

`// TODO: Get my cat on hinge so he can have a love life and get out of my hair.`